sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "geos-sys"
-version = "2.0.4"
+version = "2.1.0"  # TODO: set correct version
```

Comment:
Since we are changing the minimum supported version, it'll be a major update. But we will update the version in another PR. Please revert this change.